backend/src/main/java/de/mightypc/backend/model/specs/CPU.java

```diff
@@ -1,8 +1,17 @@
 package de.mightypc.backend.model.specs;
 
+import org.springframework.data.annotation.Id;
+
+import java.util.UUID;
+
 public record CPU(
+        @Id
+        String id,
         HardwareSpec hardwareSpec,
         int performance,
         int energyConsumption
 ) {
+    public CPU(HardwareSpec hardwareSpec, int performance, int energyConsumption){
+        this(UUID.randomUUID().toString(), hardwareSpec, performance, energyConsumption);
+    }
```

Comment:
Nutzt du den Kontruktor als deinen Hauptkonstruktor?

Ich hab mir in solchen Fällen ein Create DTO erstellt und dann einen Kontruktor dafür gebaut.

KP, ob das Good Practice ist

Reply:
Ja. Aber für tests nutze ich default konstruktor für ids